monitoring/mock_uss/run_locally_ridsp.sh

```diff
@@ -11,7 +11,13 @@ AUD=${MOCK_USS_TOKEN_AUDIENCE:-localhost,host.docker.internal}
 PORT=8071
 BASE_URL="http://${MOCK_USS_TOKEN_AUDIENCE:-host.docker.internal}:${PORT}"
 
-docker run --name mock_uss_ridsp \
+if [ "$CI" == "true" ]; then
+  docker_args="--add-host host.docker.internal:host-gateway" # Required to reach other containers in Ubuntu (used for Github Actions)
+else
+  docker_args=""
+fi
+
+docker run "${docker_args}" --name mock_uss_ridsp \
```

Comment:
Same as above:
Quotes around ${docker_args} is breaking the script (MacOS) when the value is empty.
Error: `docker: invalid reference format.`

Reply:
Added `shellcheck disable=SC2086`